README.md

```diff
@@ -1,2 +1,67 @@
-# google-chrome
-GPII Google Chrome extension
+# GPII extension for Google Chrome
+
+The GPII for Google Chrome is an extension that allows Google Chrome to be configured by the GPII and improve the accessibility of the browser in different ways.
+
+Although the extension is in development right now, it includes support for:
+* High Contrast themes
+* Enable/Disable TTS output - you need to install *ChromeVox* on your browser
+* Zoom
+
+## Building the extension
+
+Install *grunt-cli* globally:
+
+    npm install -g grunt-cli
+
+Install required dependencies:
+
+    npm install
+
+Perform a build so you can run an unpacked version of the extension on your browser.
+
+    grunt build
+
+
+If you want to create a [crx](https://developer.chrome.com/extensions/crx) package to be distributed, run:
+
+    grunt buildPkg
+
+**Note that you need to use a [PEM](http://how2ssl.com/articles/working_with_pem_files/) file to sign the crx package with. This file needs to be called *key.pem* and needs to be placed into the top level folder of this repository.**
+
+## Testing
+
+Run the node tests:
+
+    node tests/node-tests.js
+
+You can also run the browser tests, open your browser and go to:
+
+    chrome-extension://ifommhafkphfognacgcmpidnpbffedlp/tests/highContrastTests.html
```

Comment:
Well, probably I have to update the README file to indicate that, since the extension id will be different, you need to copy/paste it from the extension's section of chrome.